src/generatorprofile.cpp

```diff
@@ -1030,12 +1030,12 @@ void GeneratorProfile::GeneratorProfileImpl::loadProfile(GeneratorProfile::Profi
         mInterfaceCreateStatesArrayMethodString = "";
         mImplementationCreateStatesArrayMethodString = "\n"
                                                        "def create_states_array():\n"
-                                                       "    return [nan]*STATE_COUNT\n";
+                                                       "    return [0.0]*STATE_COUNT\n";
```

Comment:
I am concerned by this change.  It is a very good way to check the model if something isn't initialised as expected.  Can you justify this change?

Reply:
The main reason is to be in sync with what is done in C where all our arrays are (by default) initialised with zeros.

Reply:
After sleeping on it, it is our C-generated code that should be in sync with what is done in Python, and not the other way round. Ok, I am going to revert that aspect of our Python-generated code and modify our C-generated code accordingly.